### PR TITLE
Add support for multiple documents

### PR DESCRIPTION
**Pull request by so0k**
This uses `go-yaml` devel branch to iterate the yaml files for multiple documents.

It assumes the same order and prints the diff.

There's a hack in the way that I suppress `nil` output from the diff..

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -93,10 +100,28 @@ func failOnErr(formatter aurora.Aurora, errs ...error) {
...
+	return strings.Join(diffs, "\n")
+}
+
 func computeDiff(formatter aurora.Aurora, a interface{}, b interface{}) string {
 	diffs := make([]string, 0)
 	for _, s := range strings.Split(pretty.Compare(a, b), "\n") {
 		switch {
+		//hack
+		case (s == "+nil" || s == "-nil"):
+			//drop this
```
Comment (so0k) [inline main.go:124]: I don't know how else to suppress these :(

**Comment by sahilm** [comment]
Sorry for getting back after a million years. I don't think we should rely on the go-yaml dev branch. Is this feature in a released version yet?

**Comment by so0k** [comment]
it's ok, for me it was mostly just to have a way to do what I needed to do and the PR is to give others the same option

**Comment by sahilm** [comment]
Hey @so0k. After merely ~7 years I added multi-doc support as part of an experiment with Claude Code :)

**Comment by so0k** [comment]
@sahilm  - same.. I had claude code build several large scale monorepo projects
- terraform HTTP Backend (https://github.com/terraconstructs/grid)
- Blog website - from scratch in true "Cursor spirit", just powered by markdown on disk (https://github.com/terraconstructs/website)

and several yet to be public projects, have you explored SDD / Kiro yet?

**Comment by sahilm** [comment]
Nice. Good stuff. I will look into Kiro.

**Pull request closed without merging**